src/runtime.js

```diff
@@ -33,8 +35,12 @@ Object.defineProperties(Runtime.prototype, {
   module: {value: runtime_module, writable: true, configurable: true}
 });
 
-function runtime_module() {
-  return new Module(this);
+function runtime_module(define, observer = noop) {
+  if (define === undefined) return new Module(this);
+  let module = this._modules.get(define);
```

Comment:
Very interesting to use the `define` function itself as the key to the associated module...

In a sophisticated app integration context, where you perhaps have a few different notebooks that you'd like to run in the same React app (let's say), do you think we might want to refer to these different "modules" by name? Or is the define reference good enough?